lib/plugins/fs_plugin.js

```diff
@@ -22,8 +22,9 @@ module.exports = class FsWebpackPlugin {
    * @param {Object} opts.matchers - matchers from spike config
    * @param {Array} opts.module.loaders - webpack loaders from spike config
    */
-  constructor (opts) {
+  constructor (opts, util) {
     this.opts = opts
+    this.util = util
```

Comment:
I think we don't have `matchers` on the webpack config, so that may have to be added to `this.spike`